source/_remoteClient/dialogs.py

```diff
@@ -390,10 +390,11 @@ def getConnectionInfo(self) -> ConnectionInfo:
 class CertificateUnauthorizedDialog(wx.MessageDialog):
 	def __init__(self, parent: wx.Window | None, fingerprint: str | None = None):
 		# Translators: A title bar of a window presented when an attempt has been made to connect with a server with unauthorized certificate.
-		title = _("NVDA Remote Connection Security Warning")
-		message = _(
+		title = pgettext("remote", "Security Warning")
+		message = pgettext(
+			"remote",
 			# Translators: {fingerprint} is a SHA256 fingerprint of the server certificate.
```

Comment:
The translators comment could be more detailed, i.e. in addition to this useful information (or before), indicate in which situation this dialog pops up so that translators know where to check their translation if needed.